apps/system/test/marionette/lib/pinning_the_web.js

```diff
@@ -63,7 +63,9 @@ PinningTheWeb.prototype = {
   openAndPinSiteFromBrowser: function openAndPinSite(url) {
     this._openUrl(url);
     this._clickPinContextMenu();
-    // Tap the top left of the pin button.
+    // When running tests in mulet locally, part of the pin site button
+    // is cut off at the bottom of the screen. To work around this we
+    // tap the pin button in the upper left hand corner.
```

Comment:
Oh god, so lame... Still, if it works! (lame that this happens, not the work-around)